css/orbit.css

```diff
@@ -3963,22 +3963,22 @@ to use on their projects.
 	display: inline-block;
 	padding: 4px 8px;
 	border-radius: 2px;
-	font-size: 12px;
-	font-weight: 500;
+	font-size: 14px;
+	font-weight: 400;
 	text-transform: uppercase;
 }
 
 /* Type
 -------------------- */
 
 .type_info {
-	background-color: #dfe6ee;
-	color: #003055;
+	background-color: #0073FF;
+	color: #fff;
 }
 
 .type_error {
-	background-color: #ffd5d0;
```

Comment:
@elizabethkleong Were we using the wrong colors?

Reply:
Yeah they were weird greys.